tests/examples/ShoutCaseEnumMembersLinter/prefer_require_once_linter.php.in

```diff
@@ -0,0 +1,28 @@
+<?hh
+
+enum EmptyEnum: int {}
+
+enum GoodEnum: int {
+  GOOD_MEMBER = 1;
+  ALSO_GOOD_MEMBER = 1;
+}
+
+enum BadEnum: int {
+  camelCase = 1;
+  SHOUT_CASE = 1;
+  snake_case = 1;
+}
```

Comment:
Can you add some camelCaseExamplesWithMoreWords and/or numbers (especially numbers at the beginning/end of a word)?

Reply:
aNumber4InTheMiddleOf2Words
A_NUMBER4IN_THE_MIDDLE_OF2WORDS

This is definitely not very clean.

Reply:
Maybe treating a number as a capital letter in all regexes etc. would work better? Anyway it doesn't have to be perfect, it's just a suggested fix that people can always reject.

Reply:
I made numbers always break up. It is slightly better and less objectionable.

Reply:
I could rip out the `_` before or after a number, if you feel like `A_NUMBER_4_IN_THE_MIDDLE_OF_2_WORDS` is a little to excessive.

Reply:
Or I would special with a number at the end. F.e. `someCase1` would become `SOME_CASE1`, instead of `SOME_CASE_1`.

Reply:
This looks fine, I wouldn't do anything more complicated. It's probably more valuable to keep the lint rule simple, than to deal with more obscure edge cases.